packages/manager/modules/iplb/src/home/iplb-home.controller.js

```diff
@@ -369,4 +378,37 @@ export default class IpLoadBalancerHomeCtrl {
   hasMultipleRegions() {
     return isArray(this.detailedRegions) && this.detailedRegions.length > 1;
   }
+
+  showBillingEvolutionInfo() {
+    const message = this.$translate.instant(
+      'iplb_home_tile_configuration_billing_evolution_message_info',
+    );
+    const linkLabel = this.$translate.instant(
+      'iplb_home_tile_configuration_billing_more_information_label',
+    );
+    const linkURL = INFO_LINK[this.coreConfig.getUser()?.ovhSubsidiary];
```

Comment:
Please provide a fallback link
```suggestion
    const linkURL = INFO_LINK[this.coreConfig.getUser()?.ovhSubsidiary] || INFO_LINK.DEFAULT;
```
Same L:408

Reply:
Done.